### ISSUE TITLE
How can we get  GTs or annotations of published dataset?

### ISSUE THREAD

**Issue by heboyong** [David0tt/DeepGTAV#15]
In the dataset download link you posted, I can only find compressed jpeg images but not corresponding annotation files.
I would like to ask how we should obtain the annotation information of the released dataset?
Hope to get your reply as soon as possible

**Comment by David0tt** [comment]
Sorry, the annotation files must have gone missing when we converted the image files from PNGs to JPGs. We will upload the annotations and meta_data into these folders soon, in the meantime you can download them from 

https://drive.google.com/file/d/1hujK0clHEzLuCmv_SgmTmYL1s3LvNUCQ/view?usp=sharing

you must unzip them once, then copy them into the respective folderstructure and then you can extract them in their respective folders e.g. with `tar -xzf labels.tar.gz`.

**Comment by Ben93kie** [comment]
Yes, sorry, we restructured Nextcloud and removed the much bigger png-folders and only kept the jpg ones (without labels).

I reuploaded the labels and meta data. 
Please let me know if you have any issues.
